mappings/net/minecraft/item/ItemGroup.mapping

```diff
@@ -39,13 +39,15 @@ CLASS net/minecraft/class_1761 net/minecraft/item/ItemGroup
 	METHOD method_7745 setEnchantments ([Lnet/minecraft/class_1886;)Lnet/minecraft/class_1761;
 		ARG 1 targets
 	METHOD method_7747 getIcon ()Lnet/minecraft/class_1799;
-	METHOD method_7748 setNoTooltip ()Lnet/minecraft/class_1761;
+	METHOD method_7748 dontRenderName ()Lnet/minecraft/class_1761;
+		COMMENT Specifies that when this item group is selected, the name of the item group should not be rendered.
 	METHOD method_7749 setNoScrollbar ()Lnet/minecraft/class_1761;
 	METHOD method_7750 createIcon ()Lnet/minecraft/class_1799;
 	METHOD method_7751 getName ()Ljava/lang/String;
 	METHOD method_7752 isSpecial ()Z
 	METHOD method_7753 setTexture (Ljava/lang/String;)Lnet/minecraft/class_1761;
 		ARG 1 texture
-	METHOD method_7754 hasTooltip ()Z
+	METHOD method_7754 shouldRenderName ()Z
+		COMMENT Checks if this item group should render it's name.
```

Comment:
```suggestion
		COMMENT Checks if this item group should render its name.
```

Reply:
also maybe note that the name is like on top of the inventory rows, so like that "Inventory" label in the inventory screens above the player inventory slots or that inventory custom name label above the item container's inventory slots